server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/CassandraBlobModule.java

```diff
@@ -65,5 +66,12 @@
             .addPartitionKey(BucketBlobParts.ID, DataType.text())
             .addClusteringColumn(BucketBlobTable.NUMBER_OF_CHUNK, DataType.cint()))
 
+        .table(DumbBlobCache.TABLE_NAME)
```

Comment:
I would like to have this one in a separate CassandraModule, as obviously we don't always want it: it makes no sense to cache a CassandraBlobStore for instance, so cassandra-guice product will not rely on it.